lib/jazzy/sourcekitten.rb

```diff
@@ -45,7 +46,10 @@ def self.make_doc_urls(docs, parents)
 
     # Run sourcekitten with given arguments and return STDOUT
     def self.run_sourcekitten(arguments)
-      bin_path = Pathname(__FILE__).parent + '../../bin'
+      bin_path = Shellwords.shellescape Pathname(__FILE__).parent + '../../bin'
```

Comment:
Is that something that would be added into config.rb either replacing OptionParser or working with it?

I'd love to be more help, but I think from here on things go out of my 2 day glance of Ruby knowledge. So I'm now more intrigued than much help I'm afraid.